xata.ts

```
@@ -0,0 +1,29 @@
+// Generated by Xata Codegen 0.30.1. Please do not edit.
+import { buildClient } from "@xata.io/client";
+import type { BaseClientOptions } from "@xata.io/client";
+
+export type DatabaseSchema = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)